schemas/contact-schemas.js

```diff
@@ -4,6 +4,7 @@ const contactAddSchema = Joi.object({
   name: Joi.string().required(),
```

Comment:
add validation error msg, into required

validate phone and email properly, it shouldn't be just strings

Reply:
Added number and email verification.